application/src/main/scala/com/azavea/franklin/api/services/CollectionsService.scala

```diff
@@ -1,33 +1,40 @@
 package com.azavea.franklin.api.services
 
 import cats.effect._
+import cats.effect.concurrent.Ref
 import cats.syntax.all._
 import com.azavea.franklin.api.commands.ApiConfig
 import com.azavea.franklin.api.endpoints.CollectionEndpoints
 import com.azavea.franklin.api.implicits._
 import com.azavea.franklin.database.StacCollectionDao
 import com.azavea.franklin.datamodel.{CollectionsResponse, TileInfo}
 import com.azavea.franklin.error.{NotFound => NF}
+import com.azavea.franklin.extensions.validation._
 import com.azavea.stac4s._
 import doobie._
 import doobie.implicits._
 import doobie.util.transactor.Transactor
 import eu.timepit.refined.auto._
+import io.chrisdavenport.log4cats.Logger
 import io.circe._
 import io.circe.syntax._
 import org.http4s.dsl.Http4sDsl
+import sttp.client.{NothingT, SttpBackend}
 import sttp.tapir.server.http4s._
 
 import java.net.URLDecoder
 import java.nio.charset.StandardCharsets
 
 class CollectionsService[F[_]: Concurrent](
     xa: Transactor[F],
-    apiConfig: ApiConfig
+    apiConfig: ApiConfig,
+    collectionExtensionsRef: ExtensionRef[F, StacCollection]
 )(
     implicit contextShift: ContextShift[F],
     timer: Timer[F],
-    serverOptions: Http4sServerOptions[F]
+    serverOptions: Http4sServerOptions[F],
+    backend: SttpBackend[F, Nothing, NothingT],
```

Comment:
Hm, why is it an implicit param? I'd say that should be passed explicitly into the constructor.

Reply:
It's just how it works 🤷🏻‍♂️ -- `toRoutes` expects it

Reply:
@jisantuc it expects SttpBackend? o: where?